src/main/scala/org/broadinstitute/dsde/firecloud/utils/RestJsonClient.scala

```diff
@@ -44,54 +53,54 @@ trait RestJsonClient extends FireCloudRequestBuilding with PerformanceLogging {
 
   private def doRequest(addCreds: Option[RequestTransformer])(req: HttpRequest, compressed: Boolean = false, useFireCloudHeader: Boolean = false,
                                                               connector: Option[ActorRef] = None, label: Option[String] = None): Future[HttpResponse] = {
-    val sr = if (connector.isDefined) {
-      sendReceive(connector.get)(executionContext, futureTimeout = 60.seconds)
-    } else {
-      sendReceive
-    }
-
-    val pipeline = (compressed, useFireCloudHeader) match {
-      case (true, true) => addFireCloudCredentials ~> addHeader (`Accept-Encoding`(gzip)) ~> sr ~> decode(Gzip)
-      case (true, false) => addHeader (`Accept-Encoding`(gzip)) ~> sr ~> decode(Gzip)
-      case (false, true) => addFireCloudCredentials ~> sr
-      case _ => sr
+    //    val sr = if (connector.isDefined) {
+    //      sendReceive(connector.get)(executionContext, futureTimeout = 60.seconds)
+    //    } else {
+    //      sendReceive
+    //    }
+    //TODO: ask David An about the above connector code
```

Comment:
at one point, we used a [Spray host connector](http://spray.io/documentation/1.2.4/spray-can/http-client/host-level/) to throttle simultaneous requests to ontology, because we were overwhelming it: #414. But we eventually removed that code and used a different approach: #544

Reply:
If it helps to not worry about overloading ontology with a simpler approach, please do. We've moved to a k8s cluster and can spin up as many as needed to handle the volume.